terraform/validator/main.tf

```diff
@@ -8,6 +8,7 @@ locals {
 }
 
 resource "local_file" "update-validation-config" {
+  count = fileexists("${var.test_dir}/${local.validator_config}") == true ? 1 : 0
```

Comment:
Do we still need this conditional check?

Reply:
It wouldn't hurt to have that condition in place.